### PR TITLE
Entities for Level, Position, Publication, Researcher (incomplete), Student

### PR DESCRIPTION
**Pull request by cyluxtrinidad**
will work on researcher and staff entities

### CODE REVIEW

File: Spring RAP/Entities/Level.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KIT206Spring.Spring_RAP.Entities
+{
+    public enum Level
+    {
+        A,
```
Comment (cyluxtrinidad) [inline Spring RAP/Entities/Level.cs:11]: forgot to add "All"

File: Spring RAP/Entities/Student.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+namespace KIT206Spring.Spring_RAP.Entities
+{
+    public class Student : Researcher
+    {
+        // Staff Supervisor
+        public int Supervisor { get; set; }
+        public string Degree { get; set; }
+        public string SupervisorName { get; set; }
+
+        // campHouse to campus, curretn_start to current_start, lev to lvl, photURL to photoURL
```
Comment (cyluxtrinidad) [inline Spring RAP/Entities/Student.cs:16]: or can be changed to campusName

File: Spring RAP/Entities/Researcher.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+{
+    public class Researcher
+    {
+        public int ID { get; set; }
+        public ResearcherType Type { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Title { get; set; }
+        public string SchoolUnit { get; set; }
+        public Campus Camp { get; set; }
```
Comment (cyluxtrinidad) [inline Spring RAP/Entities/Researcher.cs:18]: rename to campusName

**Review by cyluxtrinidad — COMMENTED** [review]
keeping note to add "All" under public enum Level (Level.cs), Line 9 and rename campus to campusName (Researcher.cs, line 18)

**Review by Bubble-cmd — APPROVED** [review]

**Pull request merged**